apps/tangle-dapp/components/ValidatorSelectionTable/ValidatorSelectionTable.tsx

```diff
@@ -214,43 +171,28 @@ const ValidatorSelectionTable: FC<ValidatorSelectionTableProps> = ({
     () => ({
       data: allValidators,
       columns,
+      initialState: {
+        pagination: {
+          pageSize,
+        },
+      },
       state: {
         columnVisibility: {
           identityName: false,
         },
         sorting,
         rowSelection,
-        pagination,
         globalFilter: searchValue,
       },
       enableRowSelection: true,
-      onPaginationChange: setPagination,
       onGlobalFilterChange: (props) => {
-        setPagination(DEFAULT_PAGINATION);
         setSearchValue(props);
       },
       onRowSelectionChange: (props) => {
         toggleSortSelectionHandlerRef.current?.(false);
         setRowSelection(props);
       },
-      onSortingChange: (updaterOrValue) => {
-        if (typeof updaterOrValue === 'function') {
-          setSorting((prev) => {
-            const newSorting = updaterOrValue(prev);
-
-            // Modify the sorting state to always sort by the selected validators first
-            if (newSorting.length === 0) {
-              return [SELECTED_VALIDATORS_COLUMN_SORT];
-            } else if (newSorting[0].id === 'address') {
-              return newSorting;
-            } else {
-              return [SELECTED_VALIDATORS_COLUMN_SORT, ...newSorting];
-            }
-          });
-        } else {
-          setSorting(updaterOrValue);
-        }
-      },
```

Comment:
Likewise, the logic for prioritizing selected validators in the sorting process must remain intact.